src/download.rs

```diff
@@ -242,13 +245,26 @@ pub fn download_file(
     match pb {
         Some(ref pb) => {
             copy_largebuf(&mut pb.wrap_read(response), &mut buffer)?;
-            pb.finish_with_message("Done");
+            pb.finish();
         }
         None => {
             copy_largebuf(&mut response, &mut buffer)?;
         }
     }
 
+    let downloaded_length: u64;
+    if let Some(total_length) = total_length {
+        downloaded_length = total_length - starting_length;
+    } else {
+        downloaded_length = fs::metadata(dest_name).unwrap().len();
+    }
+    eprintln!(
+        "Done. {} in {} ({}/s)",
+        HumanBytes(downloaded_length),
+        FormattedDuration(starting_time.elapsed()),
+        HumanBytes(downloaded_length / starting_time.elapsed().as_secs())
```

Comment:
This breaks if downloading takes less than a second:
```
thread 'main' panicked at 'attempt to divide by zero', src/download.rs:265:20
```
Maybe if it takes that little time we shouldn't print the speed at all, because it'll be inaccurate anyway.

Reply:
Fix to calculate the speed only when the time taken is a second or more.